src/test/lib/ApiTest.cpp

```diff
@@ -692,6 +692,22 @@ void QuicTestValidateConnection()
                 &Connection.Handle));
     }
 
+    //
```

Comment:
Should we have a test for the success case just in case? It's partially covered by the existing connection tests, but only when using the current partition index, not for a different one.

Reply:
As I mentioned in the description, I don't feel it's necessary at this time. ConnectionOpen already hits every code path.